### PR TITLE
Add block coverage

### PR DESCRIPTION
**Pull request by yzhang90**

### CODE REVIEW

**Review by andrei-marinica — APPROVED** [review]

File: src/elrond-runtime.loaded.wat [hunk 1]
```diff
@@ -16,6 +16,12 @@
       <notCoveredFuncs>
         .Map
       </notCoveredFuncs>
+      <coveredBlock>
+        .Map
+      </coveredBlock>
+      <lastVisitedBlock>
+        -1
```
Comment (hjorthjort) [inline src/elrond-runtime.loaded.wat:23]: Prefer `.Int`

**Review by hjorthjort — CHANGES_REQUESTED** [review]
Looks good, but let's use `.Int` instead of `-1` where possible

File: wasm-coverage.md [hunk 2]
```diff
@@ -32,6 +34,74 @@ Function Coverage
...
+
+Block Coverage
+--------------
+
+```k
+    syntax BlockCoverage ::= blockCoverage ( blockId: Int )
+                           | ifCoverage    ( blockId: Int , truebr: Bool , falsebr: Bool )
+                           | loopCoverage  ( blockId: Int , times: Int )
+                           | "#pause"
+ // --------------------------------------------------------------------
```
Comment (hjorthjort) [inline wasm-coverage.md:47]:
```suggestion
 // ---------------------------------
```

**Review by hjorthjort — APPROVED** [review]

File: wasm-coverage.md (hunk 2 above)
Comment (hjorthjort) [reply wasm-coverage.md:47]: Actually, what does `#pause` do? I don't see it used.

**Review by hjorthjort — CHANGES_REQUESTED** [review]

**Review by hjorthjort — APPROVED** [review]

File: wasm-coverage.md (hunk 2 above)
Comment (yzhang90) [reply wasm-coverage.md:46]: used for debgging. removed

**Pull request merged**